src/app/_components/KeyMemberCard.tsx

```diff
@@ -0,0 +1,48 @@
+import Image from 'next/image'
+
+import { LinkedinLogo } from '@phosphor-icons/react/dist/ssr'
+
+import { Heading } from '@/components/Heading'
+import { Icon } from '@/components/Icon'
+import { TextLink } from '@/components/TextLink'
+
+import type { MemberData } from '@/types/memberType'
+
+type KeyMemberCardProps = MemberData
+
+export function KeyMemberCard({
+  name,
+  title,
+  linkedin,
+  image,
+}: KeyMemberCardProps) {
+  return (
+    <li className="flex rounded-lg border border-brand-500 p-1">
+      <Image
+        src={image}
+        alt={`Photo of ${name}`}
+        sizes="10rem"
+        className="aspect-[3/4] w-32 rounded object-cover"
+      />
+
+      <div className="relative m-3 grow">
+        <Heading isClamped tag="h3" variant="lg">
+          {name}
+        </Heading>
+
+        <p className="mt-1 text-brand-300">{title}</p>
+
+        <TextLink
```

Comment:
At the moment, the entire card except the image in clickable. Is this intentional?

Reply:
Ah no, good catch!

Reply:
✅